tests/functional/testHeartbeat.js

```diff
@@ -122,16 +124,15 @@ describe('Backend failure', function() {
             'VPN is on';
       });
 
-      await backendFailureAndRestore();
+      await vpn.forceHeartbeatFailure();
 
       await vpn.waitForQuery(queries.screenHome.CONTROLLER_TITLE.visible());
       assert.equal(
           await vpn.getQueryProperty(
               queries.screenHome.CONTROLLER_TITLE.visible(), 'text'),
-          'VPN is off');
+          'VPN is on');
```

Comment:
Should a comment be added here that the VPN should not be disconnected because of heart beat failure?